frontend/packages/console-app/src/components/cloud-shell/CloudShellTabHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Flex, FlexItem, FlexModifiers } from '@patternfly/react-core';
+import './CloudShellTab.scss';
+
+const CloudShellTabHeader = () => {
+  return (
+    <Flex breakpointMods={[{ modifier: FlexModifiers.grow }]}>
+      <FlexItem>
+        <span data-test-id="terminal-tab-header">Openshift command Line Terminal</span>
```

Comment:
Casing isn't consistent. Also it's spelt OpenShift.

Why a flex when there is only a single piece of text?

Reply:
I'll correct the Case of the text,
The Flex is there as it had a `RestoreLink` FlexItem to the right. Also, we'll have to add it back again.